test/hardware/TestFlashFileSystem/DemoFileSystem.c

```diff
@@ -0,0 +1,302 @@
+/*
+IMPORTANT THINGS:
+
+- each entry has 5 attributes: ID, start sector, size, isConfig and number of sectors
+- first use initFileSystem() to check if file system exits and reconstruct it
+  otherwise it just initializes all required values
+- always use findFittingStartSector() to check if your file fits. If it does, the sector to be used
+is stored in the variable nextFileSector
+- nextFileSector is the number of the sector you should write your file to. This is to be converted
+  into an actual address.
+
+*/
+
+#define SOURCE_FILE "CONFIGURE-HWTEST"
+
+#include "stdlib.h"
+#include <malloc.h>
+#include <math.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "hardware/spi.h"
+#include "pico/stdio.h"
+#include "pico/stdlib.h"
+
+#include "Common.h"
+#include "EnV5HwConfiguration.h"
+#include "EnV5HwController.h"
+#include "Esp.h"
+#include "FileSystem.h"
+#include "Flash.h"
+#include "FpgaConfigurationHandler.h"
+#include "Network.h"
+
```

Comment:
```suggestion
```

Reply:
I'm not sure why these should be removed, as they are needed.

Reply:
Only the not required empty line between the `#include` calls